### PR TITLE
Centrifuge App: Invest drawer

### PR DESCRIPTION
**Pull request by onnovisser**
### Description

New investment design.

- udates some pages to use the new page layout
- updates text and select inputs. keeps the old version around for other pages, but marked as deprecated
- placeholder button change to be closer to what the new design will be

Still todo in a follow-up:

- Add token price graph
- Add P&L values
- Improve transaction list so it doesn't scroll
- update the remaining pages to use the new page layout, once the components in the sidebar have calls to action.
- Open connect dialog when clicking invest and not connected

### Approvals

- [ ] Dev

### Screenshots

image

image

image

image

### CODE REVIEW

File: centrifuge-app/src/components/InvestRedeem/InvestRedeem.tsx [hunk 1]
```diff
@@ -449,12 +370,10 @@ function InvestForm({ onCancel, hasInvestment, autoFocus, investLabel = 'Invest'
   const isInvesting = state.pendingAction === 'invest' && isPending
   const isCancelling = state.pendingAction === 'cancelInvest' && isPending
   const isApproving = state.pendingAction === 'approvePoolCurrency' && isPending
-  const isCollecting = state.pendingAction === 'collect' && isPending
 
-  function renderInput(
-    cancelCb?: () => void,
-    preSubmitAction?: { onClick: () => void; loading?: boolean; label?: string }
-  ) {
+  console.log('state', state)
```
Comment (jpangelle) [inline centrifuge-app/src/components/InvestRedeem/InvestRedeem.tsx:374]: `console.log`

File: centrifuge-app/src/components/InvestRedeem/InvestRedeem.tsx [hunk 2]
```diff
@@ -793,76 +702,164 @@ const TransactionsLink: React.FC = () => {
...
+          </Button>
+        )}
+      </ButtonGroup>
+    </Stack>
+  )
+}
+
+function Claim({ type, onDismiss }: { type: 'invest' | 'redeem'; onDismiss?: () => void }) {
+  const { state, actions } = useInvestRedeem()
+  console.log('state.order', state.order, state.collectType)
```
Comment (jpangelle) [inline centrifuge-app/src/components/InvestRedeem/InvestRedeem.tsx:805]: `console.log`

File: centrifuge-app/src/components/Root.tsx [hunk 3]
```diff
@@ -114,7 +114,7 @@ const evmChains = {
       decimals: 18,
     },
     blockExplorerUrl: 'https://goerli.arbiscan.io/',
-    urls: ['https://arbitrum-goerli.infura.io/v3/${infuraKey}'],
+    urls: [`https://arbitrum-goerli.infura.io/v3/${infuraKey}`],
```
Comment (jpangelle) [inline centrifuge-app/src/components/Root.tsx:117]: lol 🤦

File: fabric/src/components/Select/index.tsx [hunk 4]
```diff
@@ -85,3 +112,19 @@ export const Select: React.FC<SelectProps> = ({ label, errorMessage, ...rest })
     </Stack>
   )
 }
+
+export function Select({ label, errorMessage, id, ...rest }: SelectProps) {
+  id ??= React.useId()
```
Comment (jpangelle) [inline fabric/src/components/Select/index.tsx:117]: wow, TIL

File: fabric/src/components/TextInput/index.tsx [hunk 5]
```diff
@@ -27,7 +33,119 @@ const StyledTextInput = styled.input`
...
+    color: ${({ theme }) => theme.colors.textSelected};
+  }
+`
+
+export const StyledInputBox = styled(Shelf)`
+  width: 100%;
+  position: relative;
+  background: ${({ theme }) => theme.colors.backgroundPage};
+  border: 1px solid ${({ theme }) => theme.colors.borderSecondary};
+  /* border: 1px solid transparent; */
```
Comment (jpangelle) [inline fabric/src/components/TextInput/index.tsx:62]: can remove?

**Review by jpangelle — APPROVED** [review]
nice work!! also great PR documentation 😍

**Review by sophialittlejohn — APPROVED** [review]
nice job!! 🥳

**Pull request merged**